src/seedu/addressbook/data/person/Person.java

```diff
@@ -83,5 +83,15 @@ public int hashCode() {
     public String toString() {
         return getAsTextShowAll();
     }
+    
+    String getPrintableString(Printable... printables) {
+    	String result = "";
+    	for (int i = 0; i < printables.length; i++) {
```

Comment:
a more elegant for-loop is available:

    for (Printable printable: printables) {